ragen/llm_agent/es_manager.py

```
@@ -105,13 +133,37 @@ def _expand_seed(seed: int):
             if self.mode == "train" and self.base_seed is not None:
                 self.seed_counter = seed - self.base_seed + 1
         seeds = _expand_seed(seed)
-        for seed, entry in zip(seeds, envs):
-            entry['env'].reset(seed=seed, mode=self.mode)
-            entry['status'] = EnvStatus(seed=seed)
+
+        def _reset_single(entry, single_seed):
```

Ruff (0.14.4)

Missing return type annotation for private function `_reset_single`

(ANN202)